tests/utils/test_audio_utils.py

```diff
@@ -0,0 +1,636 @@
+# coding=utf-8
```

Comment:
Very comprehensive! Wonder whether we can slim it down to 3-4 tests that have max coverage (e.g. that would tell us if we break something)? Or as Arthur suggested run it as a nightly test only (cc @ydshieh)

Reply:
There are a lot of combinations to test, so I wouldn't want to remove any of these tests.

Reply:
@ydshieh On my 2019 Intel iMac it takes 14 seconds to run these tests.

Reply:
It just loads the `"hf-internal-testing/librispeech_asr_dummy"` dataset (like the tests of most of the audio models do) but no checkpoints.